README.md

```diff
@@ -55,27 +53,46 @@ If you'd like to try it out or have any questions - feel free to join our [Slack
   
 </p>
 
-# Try out Opta
+# Features
+
+### Cloud services
+Opta supports the 3 major clouds - AWS, GCP and Azure. It has modules for the most commonly used services in these clouds like:
+* Microservices (powered by [Kubernetes](https://docs.opta.dev/architecture/aws/))
+* Databases - Postgres, MySQL, Redis
+* Serverless workloads
+* Networking - VPCs, Subnets, Load balancers
+* Cloudfront
+* Object storage (S3, GCS)
 
+Additionally, we bake in best practices like:
+* Observability (Datadog, LogDNA)
+* [SOC2 compliance](https://docs.opta.dev/compliance/)
+* [Continuous Deployment](https://docs.opta.dev/tutorials/continuous_deployment/)
+* [Hardened network and security configurations](https://docs.opta.dev/architecture/aws/)
+* Auto-scaling and high availability (HA)
 
-To use Opta, you first need to create some simple yaml configuration files that describe your needs. You can use:
 
-* Our [magical GUI](https://app.runx.dev/yaml-generator) to help generate these files.
+### Coexistence with existing Infrastructure
+Opta aims to be compatible with your existing Infrastructure setup. You can:
 
-* The [getting started](https://docs.opta.dev/getting-started/) guide and the detailed docs.
+* Import existing Terraform infrastructure into Opta
+* Use Opta outputs in Terraform files
+* Write [custom Terraform modules](https://docs.opta.dev/reference/aws/environment_modules/custom-terraform/) (for services that Opta doesn't support yet)
+* Run Opta in existing VPCs (WIP)
+
+# Try out Opta
 
-* Also, check out more [examples](https://github.com/run-x/opta/tree/main/examples)
+The best place to get started is the [Getting Started Guide](https://docs.opta.dev/getting-started/).
```

Comment:
get started - Getting Started -> Read the [Getting Started Guide]

Reply:
Can you elaborate? I don't understand what needs to be changed.

Reply:
not a big deal, but we can avoid the repetition of get started 2x

Reply:
ah fixed!